Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.10-alpine
+
+RUN pip3 install requests
+
+COPY . /opt/sapcli
+
+RUN cd /opt/sapcli &&\
+	python3 -m pip install -r requirements.txt &&\
+	chmod +x /opt/sapcli/sapcli &&\
+	echo '#!/bin/sh' > /usr/bin/sapcli &&\
+	echo 'python3  /opt/sapcli/sapcli "$@"' >> /usr/bin/sapcli &&\
```

Comment:
/sapcli is already a shell script, why don't you copy to /usr/bin/sapcli directly?

Reply:
Or you can do a symlink or you can do a symlink of ./bin/sapcli